### PR TITLE
Added --min-age and --max-age to CLI, time parsing

### PR DESCRIPTION
**Pull request by erezsh**

### CODE REVIEW

File: data_diff/parse_time.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import re
```
Comment (sirupsen) [inline data_diff/parse_time.py:1]: Can you create some unit tests for this? 🙏🏻

File: data_diff/parse_time.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    return SequenceMatcher(None, a, b).ratio()
+
+
+def parse_time_atom(count, unit):
+    count = int(count)
+    try:
+        unit = TIME_UNITS[unit]
+    except KeyError:
+        most_similar = max(TIME_UNITS, key=lambda k: string_similarity(k, unit))
+        raise ValueError(f"'{unit}' is not a recognized time unit. Did you mean '{most_similar}'?")
```
Comment (sirupsen) [inline data_diff/parse_time.py:39]: ❤️ this attention to detail!

**Comment by sirupsen** [comment]
`ValueError: Error: min_time/max_time feature requires to specify 'update_column'`

Can you include in this error to add `--update-column` if using the CLI, if you're using exceptions for user errors? 👍🏻

File: data_diff/parse_time.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    if unit in EXTRAPOLATED:
+        mul, unit = EXTRAPOLATED[unit]
+        count *= mul
+    return count, unit
+
+
+def parse_time_delta(t: str):
```
Comment (sirupsen) [inline data_diff/parse_time.py:47]: What if I pass `1h30min`? I think it'll just interpret it as `1h`, even though that's not what I meant. Probably worth erroring in this case?

Comment (sirupsen) [reply data_diff/parse_time.py:47]: What about floats? E.g. `1.5h` isn't crazy

We should probably support either floats or using multiple deltas like `1h30min`.

File: data_diff/parse_time.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
+import re
+from datetime import datetime, timedelta
+from difflib import SequenceMatcher
```
Comment (sirupsen) [inline data_diff/parse_time.py:3]: :)

**Review by sirupsen — CHANGES_REQUESTED** [review]

File: data_diff/parse_time.py (hunk 2 above)
Comment (sirupsen) [reply data_diff/parse_time.py:39]: Probably worth printing all units too

File: data_diff/parse_time.py (hunk 3 above)
Comment (erezsh) [reply data_diff/parse_time.py:47]: We already support `1h30min` ;)

File: data_diff/parse_time.py (hunk 4 above)
Comment (erezsh) [reply data_diff/parse_time.py:3]: Not sure what to do with a smiley :)

Comment (sirupsen) [reply data_diff/parse_time.py:3]: just smile!

File: data_diff/parse_time.py (hunk 3 above)
Comment (sirupsen) [reply data_diff/parse_time.py:47]: I missed the while! 👍🏻 👍🏻

**Comment by erezsh** [comment]
@sirupsen I think I addressed all your concerns

**Review by sirupsen — APPROVED** [review]

**Comment by sirupsen** [comment]
@erezsh forgot to say: can you add these to the README too?

**Comment by erezsh** [comment]
Done

**Pull request merged**